permafrost/views.py

```diff
@@ -252,6 +266,11 @@ def get_context_data(self, **kwargs):
         context['permissions'] = role.permissions().all().order_by("content_type").distinct()
         return context
 
+    def get_queryset(self):
+        qs = super(PermafrostRoleDetailView, self).get_queryset()
+        # Should be reflected in TC
+        return qs.filter(name__in=PERMAFROST_DEFAULT_ROLES)
+
 
```

Comment:
PERMAFROST_DEFAULT_ROLES will be updated to
[
'Student',
'Administrator',
'Curriculum Designer',
'Instructor',
'Site Owner'
]

Reply:
^ now will be PERMAFROST_EXCLUDED_ROLES